pkg/backend/backend.go

```diff
@@ -31,6 +33,12 @@ type Server struct {
 	rpc        spdk.JSONRPC
 	Volumes    VolumeParameters
 	Pagination map[string]int
+	psk        psk
+}
+
+type psk struct {
```

Comment:
why we need the new class here if we can just call `os.CreateTemp` and `os.WriteFile` in keyToTemporaryFile function ?

it that for unit testing ?

Reply:
Yes, for unit testing. To have an ability to stub error responses and since a file is removed at the end of the call, to check the state of that key file.

Reply:
Can't you mock the file system calls for the unit test instead of actually creating and deleting a local file?